### PR TITLE
Fix name of shopsi18 n

### PR DESCRIPTION
**Pull request by erwan-joly**

### CODE REVIEW

**Comment by erwan-joly** [comment]
unfortunately i18n file is not totally up to date. (at least in french) but good enough
fix #797

**Comment by 0Lucifer0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- src/NosCore.Controllers/Shops/MShopPacketHandler.cs  1
- src/NosCore.Parser/Parsers/NpcTalkParser.cs  1
- src/NosCore.Parser/Parsers/MapNpcParser.cs  1
- src/NosCore.Data/Dto/StaticEntities/NpcTalkDto.generated.cs  1
- src/NosCore.GameObject/ComponentEntities/Extensions/VisualEntityExtension.cs  1
- src/NosCore.GameObject/MapNpc.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5306517&bid=17366772)

File: src/NosCore.GameObject/MapNpc.cs [hunk 1]
```diff
@@ -44,12 +46,14 @@ public class MapNpc : MapNpcDto, INonPlayableEntity, IRequestableEntity, IInitia
         private readonly List<NpcMonsterDto>? _npcMonsters;
         private readonly IGenericDao<ShopItemDto>? _shopItems;
         private readonly IGenericDao<ShopDto>? _shops;
+        private List<NpcTalkDto> _npcTalks;
```
Comment (0Lucifer0) [inline src/NosCore.GameObject/MapNpc.cs:49]: Codacy Issue found: [Make '_npcTalks' 'readonly'.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5306517)

File: src/NosCore.Parser/Parsers/MapNpcParser.cs [hunk 2]
```diff
@@ -50,8 +52,8 @@ public void InsertMapNpcs(List<string[]> packetList)
             short map = 0;
             var npcs = new List<MapNpcDto>();
             var npcMvPacketsList = packetList.Where(o => o.Length > 14 && o[0].Equals("mv") && o[1].Equals("2") && long.Parse(o[2]) < 20000).GroupBy(s => s[2]).Select(s => Convert.ToInt32(s.First()[2])).ToList();
-            var effPacketsDictionary = packetList.Where(o => o[0].Equals("eff") && o[1].Equals("2") && long.Parse(o[2]) <= 20000).GroupBy(s => Convert.ToInt16(s[2])).ToDictionary(x=>x.Key, x => Convert.ToInt16(x.First()[3]));
-
+            var effPacketsDictionary = packetList.Where(o => o[0].Equals("eff") && o[1].Equals("2") && long.Parse(o[2]) <= 20000).GroupBy(s => Convert.ToInt16(s[2])).ToDictionary(x => x.Key, x => Convert.ToInt16(x.First()[3]));
+            var npcTalks = _npcTalkDao.LoadAll().ToDictionary(s => s.DialogId, s => s);
```
Comment (0Lucifer0) [inline src/NosCore.Parser/Parsers/MapNpcParser.cs:56]: Codacy Issue found: [Remove the unused local variable 'npcTalks'.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5306517)

**Comment by 0Lucifer0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- src/NosCore.Parser/Parsers/MapNpcParser.cs  1
- src/NosCore.GameObject/MapNpc.cs  2
- src/NosCore.GameObject/ComponentEntities/Extensions/VisualEntityExtension.cs  1
- src/NosCore.Data/Dto/StaticEntities/NpcTalkDto.generated.cs  1
- src/NosCore.Parser/Parsers/NpcTalkParser.cs  1
- src/NosCore.Controllers/Shops/MShopPacketHandler.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5306517&bid=17366772)

File: src/NosCore.GameObject/MapNpc.cs (hunk 1 above)
Comment (0Lucifer0) [inline src/NosCore.GameObject/MapNpc.cs:49]: Codacy Issue found: [Make '_npcTalks' 'readonly'.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5306517)

File: src/NosCore.Parser/Parsers/MapNpcParser.cs (hunk 2 above)
Comment (0Lucifer0) [inline src/NosCore.Parser/Parsers/MapNpcParser.cs:56]: Codacy Issue found: [Remove the unused local variable 'npcTalks'.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5306517)

**Comment by 0Lucifer0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           

Complexity increasing per file
==============================
- src/NosCore.Parser/Parsers/MapNpcParser.cs  1
- src/NosCore.GameObject/MapNpc.cs  2
- src/NosCore.Parser/Parsers/NpcTalkParser.cs  1
- src/NosCore.Data/Dto/StaticEntities/NpcTalkDto.generated.cs  1
- src/NosCore.GameObject/ComponentEntities/Extensions/VisualEntityExtension.cs  1
- src/NosCore.Controllers/Shops/MShopPacketHandler.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5306517&bid=17366772)

File: src/NosCore.PathFinder.Gui/PathfinderGui.cs [hunk 3]
```diff
@@ -76,7 +77,7 @@ public static void Main()
 
                 var npcMonsters = _npcMonsterDao.LoadAll().ToList();
                 TypeAdapterConfig<MapMonsterDto, GameObject.MapMonster>.NewConfig().ConstructUsing(src => new GameObject.MapMonster(npcMonsters, Logger));
-                TypeAdapterConfig<MapNpcDto, GameObject.MapNpc>.NewConfig().ConstructUsing(src => new GameObject.MapNpc(new Mock<IItemProvider>().Object, new Mock<IGenericDao<ShopDto>>().Object, new Mock<IGenericDao<ShopItemDto>>().Object, npcMonsters, Logger));
+                TypeAdapterConfig<MapNpcDto, GameObject.MapNpc>.NewConfig().ConstructUsing(src => new GameObject.MapNpc(new Mock<IItemProvider>().Object, new Mock<IGenericDao<ShopDto>>().Object, new Mock<IGenericDao<ShopItemDto>>().Object, npcMonsters, Logger, new List<NpcTalkDto>()));
```
Comment (0Lucifer0) [inline src/NosCore.PathFinder.Gui/PathfinderGui.cs:80]: Codacy Issue found: [Split this 286 characters long line (which is greater than 200 authorized).](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5306517)

**Pull request merged**